cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -309,8 +293,14 @@ def start_kube_dns():
             'dns_domain': hookenv.config('dns_domain')
         }
     }
-    create_addon('kubedns-controller.yaml', context)
-    create_addon('kubedns-svc.yaml', context)
+
+    try:
+        create_addon('kubedns-controller.yaml', context)
+        create_addon('kubedns-svc.yaml', context)
+    except CalledProcessError:
+        hookenv.status('waiting', 'Waiting to retry KubeDNS deployment')
```

Comment:
Just spotted this mistake in the failure case, should be `status_set`. I'll fix that in the rebase against upstream.